example/src/Examples/Animation/AnimateTextOnPath.tsx

```diff
@@ -53,7 +53,7 @@ export const AnimateTextOnPath = () => {
   // Create a derived value that interpolates between
   // the start and end path
   const path = useDerivedValue(
-    () => path1.interpolate(path2, progress.current),
+    () => path1.interpolate(path2, progress.current)!,
```

Comment:
These null assertion operator may feel annoying but we have `interpolatePaths` coming in another PR ;-)